src/components/EditCoursePage/index.jsx

```diff
@@ -171,6 +171,7 @@ class EditCoursePage extends React.Component {
   }
 
   formatAdditionalMetadataFields(courseData) {
+    const variant_id = courseData.additional_metadata.variant_id ? courseData.additional_metadata.variant_id : null;
```

Comment:
nit: camelCase the variable name and the conditional could probably be simplified a bit.

```jsx
const variantId = courseData.additional_metadata.variant_id || null;
return {
  // ...
  variant_id: variantId,
}
```